packages/env/lib/config/config.js

```diff
@@ -19,25 +19,25 @@ const md5 = require( '../md5' );
  * wp-env configuration.
  *
  * @typedef WPConfig
- * @property {string}                           name                    Name of the environment.
- * @property {string}                           configDirectoryPath     Path to the .wp-env.json file.
- * @property {string}                           workDirectoryPath       Path to the work directory located in ~/.wp-env.
- * @property {string}                           dockerComposeConfigPath Path to the docker-compose.yml file.
- * @property {boolean}                          detectedLocalConfig     If true, wp-env detected local config and used it.
- * @property {Object.<string, WPServiceConfig>} env                     Specific config for different environments.
- * @property {boolean}                          debug                   True if debug mode is enabled.
+ * @property {string}         name                    Name of the environment.
+ * @property {string}         configDirectoryPath     Path to the .wp-env.json file.
+ * @property {string}         workDirectoryPath       Path to the work directory located in ~/.wp-env.
+ * @property {string}         dockerComposeConfigPath Path to the docker-compose.yml file.
+ * @property {boolean}        detectedLocalConfig     If true, wp-env detected local config and used it.
+ * @property {Object.<string, WPServiceConfig>}       env                     Specific config for different environments.
```

Comment:
this seems like a weird fix. Why it is aligning the variables with the type parameter? 🤔

Reply:
Good catch @noahtallen! It's an issue with the plugin that I haven't seen before. For now, I just updated removing the space, but I'll submit a PR to the `eslint-jsdoc-plugin` with a fix in the next few days.